dds_cli/project_info.py

```diff
@@ -0,0 +1,118 @@
+
+LOG = logging.getLogger(__name__)
+
+
+###############################################################################
+# CLASSES ########################################################### CLASSES #
+###############################################################################
+
+
+class ProjectInfoManager(base.DDSBaseClass):
+    """Project info manager class."""
+
+    def __init__(
+        self,
+        project: str,
+        no_prompt: bool = False,
+        token_path: str = None,
+    ):
+        """Handle actions regarding project info in the cli."""
+        # Initiate DDSBaseClass to authenticate user
+        super().__init__(
+            no_prompt=no_prompt,
+            method_check=False,
+            token_path=token_path,
+        )
+        self.project = project
+
+    # Public methods ###################### Public methods #
+    def show_project_info(self):
+        """Get a project info."""
+        # Get info about a project from API
+        response, _ = dds_cli.utils.perform_request(
+            DDSEndpoint.PROJ_INFO,
+            method="get",
+            headers=self.token,
+            params={"project": self.project},
+            error_message="Failed to get project information",
+        )
+
+        project_info = response.get("project_info")
+
+        # Print project info table
+        table = dds_cli.utils.create_table(
+            title="Project information.",
+            columns=["Project ID", "Created by", "Status", "Last updated", "Size"],
+            rows=[
+                project_info,
+            ],
+            caption=f"Information about project {project_info['Project ID']}",
+        )
+        dds_cli.utils.console.print(table)
+
+        # Print Title and Description below the table
+        dds_cli.utils.console.print(f"[b]Project title:[/b]       {project_info['Title']}")
+        dds_cli.utils.console.print(f"[b]Project description:[/b] {project_info['Description']}")
+
+    def update_info(self, title=None, description=None, pi=None):
+        """Update project info"""
+
+        # Collect the items for change and print them before asking for confirmation
+        info_items = {}
+        if title:
+            info_items["title"] = title
+            dds_cli.utils.console.print(f"[b]New project title:[/b]         {info_items['title']}")
+        if description:
+            info_items["description"] = description
+            dds_cli.utils.console.print(
+                f"[b]New project description:[/b]   {info_items['description']}"
+            )
+        if pi:
+            info_items["pi"] = pi
+            dds_cli.utils.console.print(f"[b]New project PI:[/b]            {info_items['pi']}")
+
+        # Ask the user for confirmation
+        if not rich.prompt.Confirm.ask(
+            f"Are you sure you want to change the info for project '{self.project}'?"
+        ):
+            LOG.info("Probably for the best. Exiting.")
+            sys.exit(0)
```

Comment:
```suggestion
```

Reply:
did you change your mind about the confirmation?

Reply:
```
        # Ask the user for confirmation
        if not rich.prompt.Confirm.ask(
            f"Are you sure you want to change the title for project '{self.project}' to 

                _this_
                 to
                 _this_
            ?"
        ):
```